bin/fetch_and_pickle_schedule.py

```diff
@@ -7,16 +7,16 @@
 import os
 import pickle
 import sys
-from datetime import datetime
+from datetime import date, timedelta
+import dateutil.parser
+import pytz
 
-import pendulum
 import requests
 
 
 def from_scheduleitem_to_playout(si):
-    t = pendulum.parse(si['starttime']).in_tz('Europe/Oslo')
-    # make time naive
-    starttime = datetime(t.year, t.month, t.day, t.hour, t.minute, t.second)
+    starttime = (dateutil.parser.parse(si['starttime'])
+                 .astimezone(pytz.timezone("Europe/Oslo")))
```

Comment:
This starttime is not naive, it should be naive in Oslo time.

Reply:
Right.  I fixed it using .replace(tzinfo=None).